### PR TITLE
CI changes

### PR DESCRIPTION
**Pull request by sungam3r**
Rel: https://github.com/dotnet/format/issues/1433

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -55,9 +55,9 @@ jobs:
       - name: Check formatting
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         run: |
-          dotnet tool install -g dotnet-format --version 6.0.243104 --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json
```
Comment (sungam3r) [inline .github/workflows/test.yml:58]: Not needed anymore. Format tool is a part of NET 6 SDK.

**Comment by sungam3r** [comment]
I would make same changes in other repos.

**Comment by Shane32** [comment]
It looks like it runs the formatter three times. We can’t make it check all 3 things in one pass?

**Comment by sungam3r2022** [comment]
I'll check. Maybe today or later.

**Comment by sungam3r2022** [comment]
By the way, hi from new reality!

**Comment by sungam3r2022** [comment]
I think it is not possible now but I post an issue https://github.com/dotnet/format/issues/1540. Who knows.

**Review by Shane32 — APPROVED** [review]
Shall I merge or can you?

**Comment by Shane32** [comment]
Well this PR for sure you wouldn’t be able to

**Pull request merged**